src/NuGet.Core/NuGet.DependencyResolver.Core/Strings.resx

```diff
@@ -121,11 +121,11 @@
     <value>The feed '{0}' lists package '{1}' but multiple attempts to download the nupkg have failed. The feed is either invalid or required packages were removed while the current operation was in progress. Verify the package exists on the feed and try again.</value>
   </data>
   <data name="Log_MatchingSourceFoundForPackage" xml:space="preserve">
-    <value>Package namespace matches found for package ID '{0}' are: '{1}'.</value>
+    <value>Package source mapping pattern matches found for package ID '{0}' are: '{1}'.</value>
     <comment>{0} - Package id {1} - list of sources</comment>
   </data>
   <data name="Log_NoMatchingSourceFoundForPackage" xml:space="preserve">
-    <value>Package namespace match not found for package ID '{0}'.</value>
+    <value>Package source mapping pattern match not found for package ID '{0}'.</value>
```

Comment:
Is pattern still necessary here?

Reply:
removed the pattern from the log message